### PR TITLE
Feature/us5627 createruletables

### PR DESCRIPTION
**Pull request by lknair2015**

### CODE REVIEW

File: CI/SQL/20161116_103923_US5627_createruletables.sql [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+GO
+
+/****** Object:  Table [dbo].[cr_Rule_Genders]    Script Date: 11/16/2016 10:08:45 AM ******/
+SET ANSI_NULLS ON
+GO
+
+SET QUOTED_IDENTIFIER ON
+GO
+
+CREATE TABLE [dbo].[cr_Rule_Genders](
```
Comment (silbermm) [inline CI/SQL/20161116_103923_US5627_createruletables.sql]: make sure to check if table already exists

File: CI/SQL/20161116_103923_US5627_createruletables.sql [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+	[Domain_ID] [int] NOT NULL
+ CONSTRAINT [PK_cr_Rule_Genders] PRIMARY KEY CLUSTERED 
+(
+	[Rule_Gender_ID] ASC
+)WITH (PAD_INDEX = OFF, STATISTICS_NORECOMPUTE = OFF, IGNORE_DUP_KEY = OFF, ALLOW_ROW_LOCKS = ON, ALLOW_PAGE_LOCKS = ON) ON [PRIMARY]
+) ON [PRIMARY]
+
+GO
+
+ALTER TABLE [dbo].[cr_Rule_Genders]  WITH CHECK ADD  CONSTRAINT [FK_cr_Rule_Genders_dp_Domains] FOREIGN KEY([Domain_ID])
```
Comment (silbermm) [inline CI/SQL/20161116_103923_US5627_createruletables.sql]: each constraint will also need a guard clause (if it doesn't already exist, than alter the table)

File: CI/SQL/20161116_104923_US5627_createruleregistrationstable.sql [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+GO
+
+/****** Object:  Table [dbo].[cr_Rule_Registrations]    Script Date: 11/16/2016 10:08:45 AM ******/
+SET ANSI_NULLS ON
+GO
+
+SET QUOTED_IDENTIFIER ON
+GO
+
+CREATE TABLE [dbo].[cr_Rule_Registrations](
```
Comment (silbermm) [inline CI/SQL/20161116_104923_US5627_createruleregistrationstable.sql]: guard clause

File: CI/SQL/20161116_104923_US5627_createruleregistrationstable.sql [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+	[Domain_ID] [int] NOT NULL
+ CONSTRAINT [PK_cr_Rule_Registrations] PRIMARY KEY CLUSTERED 
+(
+	[Rule_Registration_ID] ASC
+)WITH (PAD_INDEX = OFF, STATISTICS_NORECOMPUTE = OFF, IGNORE_DUP_KEY = OFF, ALLOW_ROW_LOCKS = ON, ALLOW_PAGE_LOCKS = ON) ON [PRIMARY]
+) ON [PRIMARY]
+
+GO
+
+ALTER TABLE [dbo].[cr_Rule_Registrations]  WITH CHECK ADD  CONSTRAINT [FK_cr_Rule_Registrations_dp_Domains] FOREIGN KEY([Domain_ID])
```
Comment (silbermm) [inline CI/SQL/20161116_104923_US5627_createruleregistrationstable.sql]: guard clause

File: CI/SQL/20161116_123445_US5627_createpagesectionpagesforruleset.sql [hunk 5]
```diff
@@ -0,0 +1,49 @@
+USE [MinistryPlatform]
+GO
+
+SET IDENTITY_INSERT [dp_Pages] ON
+
```
Comment (silbermm) [inline CI/SQL/20161116_123445_US5627_createpagesectionpagesforruleset.sql]: Need a guard clause here too

File: CI/SQL/20161116_103923_US5627_createruletables.sql (hunk 1 above)
Comment (lknair2015) [inline CI/SQL/20161116_103923_US5627_createruletables.sql]: added

File: CI/SQL/20161116_103923_US5627_createruletables.sql (hunk 2 above)
Comment (lknair2015) [inline CI/SQL/20161116_103923_US5627_createruletables.sql]: added

File: CI/SQL/20161116_104923_US5627_createruleregistrationstable.sql (hunk 3 above)
Comment (lknair2015) [inline CI/SQL/20161116_104923_US5627_createruleregistrationstable.sql]: added

File: CI/SQL/20161116_104923_US5627_createruleregistrationstable.sql (hunk 4 above)
Comment (lknair2015) [inline CI/SQL/20161116_104923_US5627_createruleregistrationstable.sql]: added

File: CI/SQL/20161116_104923_US5627_createruleregistrationstable.sql (hunk 3 above)
Comment (silbermm) [inline CI/SQL/20161116_104923_US5627_createruleregistrationstable.sql]: This table is still missing an 'if not exists'

File: CI/SQL/20161116_123445_US5627_createpagesectionpagesforruleset.sql (hunk 5 above)
Comment (silbermm) [inline CI/SQL/20161116_123445_US5627_createpagesectionpagesforruleset.sql]: Still needs 'if not exists'

Comment (lknair2015) [inline CI/SQL/20161116_123445_US5627_createpagesectionpagesforruleset.sql]: added

**Pull request merged**